src/main/java/fr/rader/boblite/EditReplayTask.java

```diff
@@ -239,8 +240,10 @@ public void run() {
 
             // we open the zip file, write the recording.tmcpr file and close the zip
             replayZip.open();
-            replayZip.addFile(writer.getInputStream(), "recording.tmcpr");
+            InputStream is = writer.getInputStream();
```

Comment:
change `is` to `inputStream`, i don't like acronyms for variable names

Reply:
Input stream name is now changed, d469b30